src/main/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientImpl.java

```diff
@@ -183,7 +192,9 @@ public GraphqlClientConfiguration getConfiguration() {
             String json;
             try {
                 json = EntityUtils.toString(entity, StandardCharsets.UTF_8);
+                stopTimer.run();
```

Comment:
Please add to the Wiki page, that the request duration will only consider successful HTTP requests.

Reply:
Does it make sense to also measure unsuccessful ones? Maybe it does, if so, do we need to separate the timers by response code then? This would actually be difficult because we don't know the resp. code when we start the timer. But lets assume the backend times out internally and returns a 500 after 60s. That would distort the success timer metric noticeably.

Reply:
IMO it does not make sense, exactly because of the distortion. We track error rates anyway, so we can see when something is wrong. 

Just to make this clear, because there might be people that expect timeouts to go in the request duration as well, we should clarify that on the wiki page.